### PR TITLE
Also resolve refs in additionalProperties

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

File: openapi2conv/openapi2_conv.go [hunk 1]
```diff
@@ -242,6 +242,9 @@ func ToV3SchemaRef(schema *openapi3.SchemaRef) *openapi3.SchemaRef {
 	for k, v := range schema.Value.Properties {
 		schema.Value.Properties[k] = ToV3SchemaRef(v)
 	}
+	if schema.Value.AdditionalProperties != nil {
```
Comment (fenollp) [inline openapi2conv/openapi2_conv.go]: `AdditionalProperties` can also be a boolean.

Comment (chirino) [reply openapi2conv/openapi2_conv.go]: What do you mean? It's defined as `AdditionalProperties *SchemaRef`
Do you mean AdditionalPropertiesAllowed??

Comment (chirino) [reply openapi2conv/openapi2_conv.go]: What change are you requesting?

Comment (fenollp) [reply openapi2conv/openapi2_conv.go]: Oh you're right I forgot that this is handled with multijson tags.

**Pull request merged**